rln-js/index.html

```diff
@@ -176,6 +188,118 @@
         updateMessages(messages, messagesDiv)
     }
 
+
+    const rlnDeployBlk = 7109391;
+    const rlnAddress = "0x4252105670fe33d2947e8ead304969849e64f2a6";
```

Comment:
I think it needs to be updated see https://discord.com/channels/@me/824090196073119775/1024939970143727656